app/src/main/java/one/mixin/android/db/MessageDao.kt

```diff
@@ -562,11 +548,12 @@ interface MessageDao : BaseDao<Message> {
     suspend fun deleteMessageByConversationId(conversationId: String, limit: Int)
 
     @Transaction
-    fun insertAndNotifyConversation(message: Message, conversationDao: ConversationDao, userId: String?) {
+    fun insertAndNotifyConversation(message: Message, remoteMessageStatusDao: RemoteMessageStatusDao) {
         insert(message)
-        if (userId != message.userId) {
-            conversationDao.unseenMessageCount(message.conversationId, userId)
+        if (!message.isMine()) {
+            remoteMessageStatusDao.insert(RemoteMessageStatus(message.id, message.conversationId, MessageStatus.DELIVERED.name))
         }
+        remoteMessageStatusDao.updateConversationUnseen(message.conversationId)
```

Comment:
Maybe we should only update conversation's unseen count when inserting someone else's message.

Reply:
https://github.com/MixinNetwork/android-app/commit/e32fda408432c36d452bd262b995193735769d73